### DISCUSSION TITLE
Support for Filters in Chart and Stats Widgets

### DISCUSSION THREAD

**Discussion by ManojKiranA** [filamentphp/filament#5588] [Ideas]
Based on the discord [showcase](https://discord.com/channels/883083792112300104/883084832387760148/1069985046661971968) by @leandrocfe  for [leandrocfe/filament-apex-charts](https://github.com/leandrocfe/filament-apex-charts) 

- Chart widget only support `single` dropdown filter. It would be nice to have option to add multiple Filters in chart widgets
- Stats filter doesn't have any filters at all. Adding dropdown filter would be a nice addition

**Comment by ManojKiranA** [comment]
Implemented in v3

https://filamentphp.com/docs/3.x/panels/dashboard#filtering-widget-data

**Reply by BoomCheeseBurge** [reply]
that is not what i expected at all...
